.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
-  "arrowParens": "always",
+  "arrowParens": "avoid",
```

Comment:
Did you deem Prettier necessary for this to work? `.toMatchSnapshot` did not work as intended?

Reply:
I have changed to toMatchSnapshot